types/notifyDependencyPaths.ts

```diff
@@ -0,0 +1,10 @@
+export const NOTIFY_DEPENDENCY_PATHS_NOTIFICATION_METHOD = 'aws/notifyDependencyPaths'
+
+export interface NotifyDependencyPathsParams {
+    moduleName: string
+    programmingLangugage: string
```

Comment:
typo. Also, "prog lang" may be misleading. This is more like a "mode" or "kind" which triggers heuristics that look at the workspace. In other places (Lambda, SAM CLI, Toolkits, even [CodeWhisperer](https://github.com/aws/aws-toolkit-vscode/blob/cc0455f112c3be8db3232acfb777341f3225c557/packages/core/src/codewhisperer/util/runtimeLanguageContext.ts#L16)), it's called a "runtime".

For example, "node" is not a programming language, but a node project involves js + ts. Which "programming language" would we specify there?

```suggestion
    runtime: string
```

Reply:
That's a good point, what about `runtimeLanguage` I still think we need the language information and not the runtime

For a node project with both js and ts files, the client can send one notification for each language, imo this is not redundant but rather being explicit in the communication. I am open to discuss